WalletWasabi.Fluent/Controls/TagsBox.axaml

```diff
@@ -28,7 +28,7 @@
     <Setter Property="BorderBrush" Value="{DynamicResource TextControlBorderBrush}" />
     <Setter Property="BorderThickness" Value="{DynamicResource TextControlBorderThemeThickness}" />
     <Setter Property="FontSize" Value="{DynamicResource ControlContentThemeFontSize}" />
-    <Setter Property="MinHeight" Value="{DynamicResource TextControlThemeMinHeight}" />
+    <Setter Property="MinHeight" Value="71" />
```

Comment:
@ichthus1604 I would prefer using the same height as the TextControl because we used to have `TextBox` and `TagsBox` on the same page so it can also happen in the future. Because of the aesthetic look, it would be nice if both controls have the same height.

I believe the source if the issue is the `Margin` of the `TagControl` items. How about using negativ margin instead?
```xml
<Style Selector="c|TagsBox c|TagControl">
  <Setter Property="Margin" Value="0,-3,4,0" />
</Style>
```